<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NLogin</class>
 <widget class="QWidget" name="NLogin">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>450</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,3">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>25</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <property name="horizontalSpacing">
         <number>15</number>
        </property>
        <property name="verticalSpacing">
         <number>30</number>
        </property>
        <item row="0" column="1">
         <widget class="QLineEdit" name="n_useNameLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>Admin</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 75 25pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>密码:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 75 25pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>用户:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="n_usePwdLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;AcadEref&quot;;</string>
          </property>
          <property name="text">
           <string>888888</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>85</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/icon/loginButton-a.png);
border-image: url(:/icon/loginButtontwo.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
